src/homepageExperience/ClickThroughAnnouncementHandler.tsx

```diff
@@ -0,0 +1,67 @@
+// Libraries
+import {FC, useEffect} from 'react'
+import {useSelector, useDispatch} from 'react-redux'
+import {useLocalStorageState} from 'use-local-storage-state'
+
+// Selectors
+import {selectCurrentIdentity} from 'src/identity/selectors'
+
+// Utils
+import {showOverlay, dismissOverlay} from 'src/overlays/actions/overlays'
+
+// Constants
+import {CLOUD} from 'src/shared/constants'
+
+export const ClickThroughAnnouncementHandler: FC = () => {
+  const dispatch = useDispatch()
+  const {account} = useSelector(selectCurrentIdentity)
+
+  const [announcementState, setAnnouncementState] = useLocalStorageState(
+    'clickThroughAnnouncement',
+    {
+      announcementID: '',
+      display: true,
+    }
+  )
```

Comment:
By using `useLocalStorageState`, users who already acknowledged the click through announcement will not see the announcement again. However, users will see the announcement again if they do the following:
* clear their browser cache
* use another browser
* use a private browser

Is it right?

Reply:
Yes, this is correct and is acceptable behavior for these types of announcements.